lib/mongoid/association/embedded/embeds_many/proxy.rb

```diff
@@ -236,7 +236,7 @@ def exists?
           #   person.addresses.find { |addr| addr.state == 'CA' }
           #
           # @param [ Object... ] *args Various arguments.
-          # @param [ Proc ] block Optional block to pass.
+          # @yield &block Optional block to pass.
```

Comment:
For example, here, the block is a parameter and should be documented as `@param`. There may additionally be a `@yield` clause stating the method yields to the passed block, with `@yieldparam` listing the parameters yielded, if any.